docs/benchmark/ftlib_benchmark.md

```diff
@@ -0,0 +1,131 @@
+## Benchmark of FTlib AllReduce
+
+We perform experiments to test the performance of FTlib AllReduce in
+ElasticDL on Minikube, the CPU cluster, and GPU cluster. We adopt
+ResNet50 and MobileNetV2 in experiments. ResNet50 is a computation-intensive
+model and MobileNetV2 is a communication-intensive model.
+
+## Minikube
+
+The setups of experiments are:
+
+| Experiment parameter | Value    |
+| -- | --- |
+| Batch size | 64 |
+| Batches per task | 50 |
+| Images per task | 3220 |
+| Dataset | cifar10 |
+| Image shape | (32, 32, 3) |
+| Worker resource| cpu=0.3,memory=2048Mi,ephemeral-storage=1024Mi|
+
+### ResNet50
+
+The number of trainable parameters in ResNet50 for cifar10 is 23,555,082.
```

Comment:
Do you mean you used the CIFAR 10 dataset to train the ResNet50 model in this experiment?

Reply:
Yes. I will polish the content.